### PR TITLE
Add spec for Angular baseline implementation (Hippopotamus) (GSI-1086)

### PR DESCRIPTION
**Pull request by Cito**

### CODE REVIEW

File: 60-hippopotamus/technical_specification.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The goal of this epic is to create a repository that can be used as a scaffold (baseline) for the re-implementation of the GHGA Data Portal in Angular.
+
+The scaffold should follow the decisions that we made in the [Bottlenose Dolphin](../59-bottlenose-dolphin/technical_specification.md) epic.
+
+The scaffold application should already implement the proper overally visual layout and design (styling and theming). It should be runnable against the mock backend and the real backend and should exemplify using at least one REST endpoint (e.g. the global stats shown on the homepage). It should cover all tooling to provide a good developer experience, linting and running unit and e2e testes.
+
+### Included/Required:
+
+- Create a new GitHub repository "data-portal" from scratch that will eventually supersede the existing "data-portal-ui" repository.
+- The experimental repository "angular-portal" will be only used as reference and then archived.
```
Comment (SilverLinings89) [inline 60-hippopotamus/technical_specification.md:21]:
```suggestion
- The experimental repository "angular-portal" will only be used as a reference and then archived.
```

File: 60-hippopotamus/technical_specification.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Add pre-commit hooks and configuration for Prettier, ESLint and other useful linters.
+- Add `@angular-eslint/eslint-plugin` with the recommended rules as specified in [ADR013](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr013_angular_code_style.md).
+- Add subdirectories as specified in [ADR018](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr018_frontend_architecture.md).
+- Add `eslint-plugin-boundaries` configuration as specified in [ADR018](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr018_frontend_architecture.md).
+- Add and configure ESLint plugin to enforce documentation via JSDoc as specified in [ADR014](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr014_angular_project_documentation.md).
+- Add support for Compodoc as specified in [ADR014](https://github.com/ghga-de/adrs/blob/main/docs/adrs/).
+- Add the latest Angular Material version as dependency as specified in [ADR019](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr019_responsive_design_systems.md) and [ADR020](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr020_angular_component_library.md).
```
Comment (SilverLinings89) [inline 60-hippopotamus/technical_specification.md:32]:
```suggestion
- Add the latest Angular Material version as a dependency as specified in [ADR019](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr019_responsive_design_systems.md) and [ADR020](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr020_angular_component_library.md).
```

File: 60-hippopotamus/technical_specification.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Create the base components (app component, header, footer, landing page) and style them according to our existing corporate design, make everything look similar to the legacy application.
+- Make sure the application uses the proper semantic tags according to [ADR016](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr016_semantic_web_technologies.md).
+- Add a minimal multi-stage production docker file that uses the run script to inject the configuration without rebuilding the application and SWS (`static-web-server`) to serve it as a single-page application.
+- Add and configure `cashew` for caching HTTP requests as a dependency.
+- Add and configure `MSW` (Mock Service Worker) for mocking the backend as a dependency.
+- Add authentication (login via LS Login, registration, 2FA) using `oidc-client-ts` (will be moved to the backend later). The profile page can be incomplete.
+- Components that require authentication should be guarded and lazy-loaded.
+- Make sure it is easy to switch between testing the application manually on localhost against MSW and against the staging or testing backend deployment.
```
Comment (SilverLinings89) [inline 60-hippopotamus/technical_specification.md:42]: This is a bit vague. I think the most important part of this is to have it documented in a way that makes it easy to follow.

File: 60-hippopotamus/technical_specification.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Remove Karma and Jasmine and add Jest as well as `jest-preset-angular` and `jest-marbles` as dependencies for unit testing. 
+- Add some example unit tests using Jest.
+- Add and configure Playwright for e2e-testing. These tests only need to work with MSW. Later we may also add e2e-tests against the testing deployment in this repo, but they could be also added to the Archive Test Bed instead.
```
Comment (SilverLinings89) [inline 60-hippopotamus/technical_specification.md:45]:
```suggestion
- Add and configure Playwright for e2e-testing. These tests only need to work with MSW. Later we may also add e2e-tests against the testing deployment in this repo, but they could also be added to the Archive Test Bed instead.
```

File: README.md [hunk 5]
```diff
@@ -67,3 +67,4 @@ This repository is part of the [Epic Planning and Marathon SOP](https://docs.ghg
 - (57) [Monkfish](./57-monkfish/technical_specification.md): State Management Service
 - (58) [Poodle Moth](./58-poodle-moth/technical_specification.md): Storing Data as strings vs BSON in MongoDB
 - (59) [Bottlenose Dolphin](./59-bottlenose-dolphin/technical_specification.md): Angular implementation plan and migration strategy
+- (60) [Hippopotamus](./60-hippopotamus/technical_specification.md): Create scaffold for Angular implementation
```
Comment (SilverLinings89) [inline README.md:70]:
```suggestion
- (60) [Hippopotamus](./60-hippopotamus/technical_specification.md): Create basic Angular implementation
```
I think with the given scope, it's not just a scaffold for an implementation. It does contain more than that.

**Review by SilverLinings89 — DISMISSED** [review]
Only minor comments, looks good to me!

File: 60-hippopotamus/technical_specification.md (hunk 3 above)
Comment (Cito) [reply 60-hippopotamus/technical_specification.md]: True, improved in b100db27.

File: README.md (hunk 5 above)
Comment (Cito) [reply README.md]: Ok, let's call it "baseline application". Changed in cba521d7.

**Review by ac-jorellanaf — APPROVED** [review]

**Review by SilverLinings89 — APPROVED** [review]

**Pull request merged**